packages/js/e2e-environment/package.json

```diff
@@ -35,7 +35,8 @@
     "jest-each": "25.5.0",
     "jest-puppeteer": "^4.4.0",
     "node-stream-zip": "^1.13.6",
-	"readline-sync": "^1.4.10",
+    "puppeteer": "2.1.1",
```

Comment:
The addition of the `puppeteer` dependency seems to be causing an issue with the CI. I committing the `pnpm-lock.yaml` file from running `pnpm i` in the `plugins/woocommerce` folder may do the trick.

Reply:
Thanks @MrJnrman, it passed this time.

Reply:
Thanks for making the change @rrennick